src/mito2/src/worker.rs

```diff
@@ -367,7 +367,7 @@ impl<S: LogStore> RegionWorkerLoop<S> {
     }
 }
 
-impl<S> RegionWorkerLoop<S> {
+impl<S: LogStore> RegionWorkerLoop<S> {
```

Comment:
Since we need `S: LogStore`, we can move this method to the `impl block` above.
```rust
impl<S: LogStore> RegionWorkerLoop<S> {
    async fn handle_requests(&mut self, buffer: &mut RequestBuffer) {}

    async fn handle_ddl_requests(&mut self, ddl_tasks: Vec<RegionTask>) {}
}

impl<S> RegionWorkerLoop<S> {
    async fn clean(&self) {}
}
```